receiver/jaegerlegacyreceiver/go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jaegerlegacyreceiver
+
+go 1.12
```

Comment:
Any reason to keep this version? Other components are not setting a specific version.

Reply:
Done - although the other receivers all have versions defined (some 1.12, others 1.13).